solutions/devsprint-attekita-3/FinanceAppTests/Service/StringExtensionTest.swift

```diff
@@ -0,0 +1,23 @@
+//
+// StringExtensionTest.swift
+//  FinanceAppTests
+//
+//  Created by Tatiana Rico on 12/05/22.
+//
+import XCTest
+@testable import FinanceApp
+
+class StringExtensionTest: XCTestCase {
+
+    func test_remove_space_String_Text_dont_have_Space() {
+        let textOneSpace = "Dia Lindo"
+        let textTwoSpace = "Dia a dia"
+        let text2TwoSpace = "Algo  Aconteceu"
+        
+        XCTAssertEqual(textOneSpace.removingSpaces(), "DiaLindo")
+        XCTAssertEqual(textTwoSpace.removingSpaces(), "Diaadia")
+        XCTAssertEqual(text2TwoSpace.removingSpaces(), "AlgoAconteceu")
+    }
+
```

Comment:
Poderíamos escrever mais alguns cenários de teste:

- Long spaces
O que acontece quando passo uma string como essa? 
"               olá" ( o GitHub esta tirando os espaços  mas a ideia era ter um grande espaço na string"

- E se a minha string for vazia?
""

Se tivéssemos um algoritmo com problemas, certamente seria acusado em um caso desses
Esses testes você pode fazer em casos de testes diferentes :)